app/src/ui/thank-you/thank-you.tsx

```diff
@@ -107,7 +104,7 @@ export class ThankYou extends React.Component<IThankYouProps, {}> {
       >
         <DialogContent>
           <div className="container">
-            <div className="thank-you-note">{thankYouNote}.</div>
```

Comment:
This existed previously to facilitate the conditional addition of the version number, so I removed it here because with the second sentence we'll need to do it a bit differently.